src/universal/modules/userDashboard/containers/UserSettings/UserSettingsContainer.js

```diff
@@ -76,7 +88,56 @@ export default class UserSettingsContainer extends Component {
       router.push(nextPage);
     }
     untouch('preferredName');
-  };
+  }
+
+  uploadPicture(pictureFile) {
```

Comment:
OK i still wanna try n poke some holes in this--

Currently, the client has to:
- send a request to Parabol for the `signedPutUrl`
- wait for the `signedPutUrl` to come back from Parabol
- send the picture to amazon
- wait for success to come back from amazon
- send updated picture url to Parabol
- wait for new url from Parabol

if we do send the picture to Parabol instead of S3, then we avoid 2 client network requests, which is kinda a big deal, assuming that they have a longer distance to travel than Parabol -> AWS. Granted, at the expense of extra data being received by our server. If you're happy I'm happy, just wanted to point this out.